_posts/2019-12-03-ios-learning-group.markdown

```diff
@@ -0,0 +1,117 @@
+The book is written for people who want to improve their own learning skills, but it was _very_ helpful to read as
+I was developing and delivering this curriculum. Here are a few lessons that I learned from the book that were
+helpful while teaching engineers at Artsy about how we build iOS software:
+
+- If someone tries to do something themselves _before_ being told how to do it, the attempt will strengthen their
+  understanding of the underlying concept. To put this into practice, I would often ask learners questions that I
+  didn't expect they could answer yet, and the resulting discussion was always worthwhile.
+- Interleaving different concepts together helps learners form connections between those concepts. This was
+  especially important, since a big motivator for using React Native at Artsy was to share skills between web and
+  iOS codebases. As an example of putting this into practice, I interleaved a discussion of
+  [Relay](https://relay.dev) into our curriculum; I hoped to show learners both a new perspective of Relay, as well
+  as show them how familiar writing React Native code was to writing React web code.
+- Allowing for some forgetting to take place before reviewing concepts will help strengthen learner's
+  understanding. To put this into practice, I would return to topics from a few weeks ago to cement their
+  understanding with learners.
+
+I had to push through some discomfort as an educator, too. Each session ended with homework questions, which we
+reviewed at the top of the next session. I would ask each question and then just sit there, in awkward silence,
+while everyone looked around for someone to answer. Eventually, inevitably, someone would.
+
+All of the learning materials
+[are open source](https://github.com/artsy/README/tree/master/resources/mobile/learning-group). While the materials
+are mostly specific to Artsy, they may be of help to others. And regardless, we want to adhere to our
+[Open Source by Default](https://github.com/artsy/README/blob/master/culture/engineering-principles.md#open-source-by-default)
+principle.
+
+We learned quite a lot from delivering this curriculum – lessons we can apply to our next learning group:
+
+- Learners appreciated the weekly schedule set upfront, affirming what we learned from previous learning groups.
+- Learners appreciated having access to the materials ahead of time.
+- Learners appreciated having the sessions recorded, to be reviewed later (or watched, in case they missed the
+  session).
+- Learners appreciated having office hours available; even though the office hours weren't well-attended, learners
+  appreciated having access to them if they needed to.
+- Learners even appreciated the awkward silences while I waited for an answer to my questions. (One survey
+  respondent described it as "like pulling teeth, but helpful.")
+- Learners are varied in how they want to learn. Some liked going through things together. Some thought we went too
```

Comment:
I'm in the process of reading "Make It Stick" right now, and I remember a chapter that talked about different styles of learning. My takeaway from the chapter was to push people toward the learning styles recommended by the book, as "learning styles" often means "what I am comfortable with, regardless of how effective it may be". I'm curious (a) if you remember that chapter, (b) you had different takeaways, and (c) if it affects how you think about this last bullet point.

Reply:
Hmm! I do remember, and I agreed with it (from my own learning experiences). I think there's room on this last bullet point to differentiate learning _styles_ (auditory, visual, etc) from the activities we do to learn (review material beforehand? etc). I think there's room on the latter, to support both people who want to come prepared and those who don't have the time. I'll revisit 👍

Reply:
This is worth calling out, but I don't want to distract either. I've added a paragraph below that is sufficiently hand-wave-y to acknowledge this, without diving too deeply. Thoughts?

Reply:
I'm sorry if you lost cycles on this - I thought I edited my comment to include a statement that I wasn't expecting any changes to the article based on my comment. What you added is great, though!